src/cli/Clip.cpp

```diff
@@ -32,11 +32,16 @@ const QCommandLineOption Clip::TotpOption = QCommandLineOption(QStringList() <<
                                                                              << "totp",
                                                                QObject::tr("Copy the current TOTP to the clipboard."));
 
+const QCommandLineOption Clip::UsernameOption = QCommandLineOption(QStringList() << "u"
+                                                                                 << "username",
+                                                               QObject::tr("Copy the current username to the clipboard."));
+
 Clip::Clip()
 {
     name = QString("clip");
-    description = QObject::tr("Copy an entry's password to the clipboard.");
+    description = QObject::tr("Copy an entry's password or username to the clipboard.");
```

Comment:
@sjamesr this description is already outdated, since we allow the current totp to be copied. Maybe we should update it to `Copy an entry's field`?